LICENSE

```diff
@@ -236,19 +236,35 @@ Refer to the above license for the full text.
 
 This product bundles JSON formatter, which is available under an Apache-2.0 license.
 @traffic_portal/app/src/assets/css/jsonformatter.min_0.6.0.css
```

Comment:
We could put `@traffic_portal/app/src/assets/*/jsonformatter.min_0.6.0.*` and match both with one line.

Reply:
Globbed in 5fb856767d